Source/WebCore/accessibility/AccessibilityObject.h

```diff
@@ -745,6 +745,8 @@ class AccessibilityObject : public AXCoreObject, public CanMakeWeakPtr<Accessibi
 
     const AccessibilityScrollView* ancestorAccessibilityScrollView(bool includeSelf) const;
     virtual AccessibilityObject* webAreaObject() const { return nullptr; }
+    virtual bool shouldWebAreaExposeChildren() const { return true; }
```

Comment:
Is this a webarea specific property? I.e., can it apply to other types of objects?

Reply:
No, this is for web areas only since the children will ask for this on their web area. However, I am going to rename this per Tyler's suggestion above.